### PR TITLE
retool-rpc v0

### PR DESCRIPTION
**Pull request by mikewang333**
walkthrough of using rpc in cli:
https://www.loom.com/share/7845c044752c4146b40a351fba091e46

### CODE REVIEW

File: src/utils/credentials.ts [hunk 1]
```diff
@@ -164,3 +164,19 @@ export async function getAndVerifyFullCredentials() {
   spinner.stop();
   return credentials;
 }
+
+export function getAndVerifyCredentials() {
```
Comment (mikewang333) [inline src/utils/credentials.ts:168]: It seems like the other function is more specific to accessing retoolDB so just made another one to verify Retool credentials. lmk if that's ok @PeteTheHeat

File: src/commands/rpc.ts [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+      name: "destinationPath",
+      message: "Where would you like to create this app?",
+      type: "input",
+      default: "./" + "retool_rpc" + "_" + appType,
+    },
+  ])) as { destinationPath: string };
+
+  const queryResult = await createPlaygroundQuery(resourceId, credentials);
+
+  console.log("Creating app...");
```
Comment (PeteTheHeat) [inline src/commands/rpc.ts]: Nit: Move above `createPlaygroundQuery` ?

Nit: the logging pattern I've been following for potentially long, async tasks is to 

```
const spinner = ora("Creating app").start();

// async stuff

spinner.stop()
```

**Review by PeteTheHeat — APPROVED** [review]
Nice!

File: src/commands/rpc.ts (hunk 2 above)
Comment (mikewang333) [reply src/commands/rpc.ts]: good catch! used the spinner and started it before creating the playground query :)

**Pull request merged**